### ISSUE TITLE
[Accessibility] Screen reader incorrect reading of SfTextInputLayout

### ISSUE THREAD

**Issue by tj-devel709** [syncfusion/maui-toolkit#171]
### Description

There seems to be an issue when the screenreader on at least Catalyst and Windows focuses on a SfTextInputLayout that contains a Picker. Using syncfusion maui-toolkit version 1.0.30, I can focus on a SFTextInputLayout that has a picker and interact with it. When using VoiceOver on Catalyst on 1.0.40, it just skips over the SFTextInputLayout entirely. On Windows, it also skips over it completely on 1.0.40 but works in 1.0.30.

Below shows the correct behavior of the picker in 1.0.30

Below shows the incorrect behavior of the picker in 1.0.40. Note that usually using the control+Option+ArrowKeys when VoiceOver is on, it skips over the picker. It should not do this. I can get to the picker using the tab button but then it just does some random stuff and I cannot interact with it anyways.

### Steps to Reproduce

1. Clone the maui-samples repo on the dev/TJ/SFTest_Picker branch - https://github.com/dotnet/maui-samples/tree/dev/TJ/SFTest_Picker/9.0/Apps/DeveloperBalance 
2. Open the 9.0/Apps/DeveloperBalance app
3. Click any project and go to the ProjectDetails Page
4. Turn on the screenreader and try to access the Category Picker using the keyboard  

### Version with bug

1.0.4

### Is this a regression from previous behavior?

Yes, this used to work

### Last Known Working Version

1.0.3

### Affected platforms

macOS, Windows

### Have you found a workaround?

No workaround found

### Relevant log output

```shell

```

**Comment by tj-devel709** [comment]
Actually having trouble uploading the videos right now
